doc/reference/files/dune-project/dialect.rst

```diff
@@ -65,3 +65,9 @@ dialect
       This field supports the same sub-fields as ``implementation``.
 
       .. versionchanged:: 3.9 This field is made optional.
+
+Dune comes with some dialects pre-set and enabled. The most common one is
```

Comment:
Thanks. I think this should go into a new "Default Sialects" section.

Also, we're in a reference section so we should be precise: the default dialects are exactly `ocaml` and `reason`. In addition, if `(using melange)` is used, the `rescript` dialect is enabled (I just learned this!). Maybe this should be mentioned.

Reply:
Good point, I've added these. I think switching to an enumeration of dialects works better than a freeform paragraph as it was before.